packages/graphql/jest.config.js

```diff
@@ -1,20 +1,13 @@
-const pkg = require('./package');
+// For a detailed explanation regarding each configuration property, visit:
+// https://jestjs.io/docs/en/configuration.html
 
 module.exports = {
-  rootDir: './',
```

Comment:
Are these changes really necessary?

Reply:
Yes, due to the nature of the new architecture proposal there were I few things that needed to be changed in order to adjust to this architecture.